designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/Statistics.scala

```diff
@@ -23,6 +26,9 @@ object Statistics extends LazyLogging {
       encodeQueryParam(RequestIdStat.name -> requestId.value)
     )
 
+    private val encryptedQueryParam = "encrypted="
+    private val keyQueryParam       = "&key="
+
     override def prepareURLs(cfg: StatisticUrlConfig): Either[StatisticError, List[URL]] =
       rawStatistics.toList
         // Sorting for purpose of easier testing
```

Comment:
If we are going to encrypt data we should consider changing size limit because after encryption data is longer